### DISCUSSION TITLE
Sequence diagram: alternative / if-else condition

### DISCUSSION THREAD

**Discussion by YosypC** [gaphor/gaphor#1730] [Q&A] [answered]
New to the program and learning UML, I can't find the [alternative](https://stackoverflow.com/questions/8114770/how-to-show-if-condition-on-a-sequence-diagram) box in Gaphor. Where is it?

EDIT: also, what about loops and other "fragments"?

**Comment by amolenaar** [comment, marked as answer]
Hi @YosypC,

Currently sequence diagrams in Gaphor do not support fragments. As an alternative you can use comments for now.

Do you want to create an issue for adding fragments, so we do not forget?

**Reply by YosypC** [reply]
[Done.](https://github.com/gaphor/gaphor/issues/1731)
I'd really help you but:

- I don't know UML
- I have extremely limited experience in Python, and currently can't dedicate time to it
Wish you the best!
